libs/net/dal/Services/ContentService.cs

```diff
@@ -62,6 +62,7 @@ public IPaged<Content> FindWithDatabase(ContentFilter filter, bool asNoTracking
             .Include(c => c.Source)
             .Include(c => c.Series)
             .Include(c => c.License)
+            .Include(c => c.TonePoolsManyToMany).ThenInclude(ct => ct.TonePool)
```

Comment:
While helpful, I don't think this is going to help.  We shouldn't be using this for search from the frontend anymore.  We're using elasticsearch.  The method below this one.